pool/hub.go

```diff
@@ -442,10 +445,13 @@ func (h *Hub) getWork(ctx context.Context) (string, string, error) {
 
 // getTxConfNotifications streams transaction confirmation notifications for
 // the provided transaction hashes.
-func (h *Hub) getTxConfNotifications(txHashes []*chainhash.Hash, stopAfter int32) (func() (*walletrpc.ConfirmationNotificationsResponse, error), error) {
+func (h *Hub) getTxConfNotifications(txHashes []chainhash.Hash, stopAfter int32) (func() (*walletrpc.ConfirmationNotificationsResponse, error), error) {
 	hashes := make([][]byte, 0, len(txHashes))
+	log.Tracef("Requesting tx conf notifications for %d "+
+		"transactions (stop after #%d)", len(txHashes), stopAfter)
```

Comment:
Checking docs for `stopAfter` at https://github.com/decred/dcrwallet/blob/master/rpc/documentation/api.md#confirmationnotifications

> `int32 stop_after`: The upper limit on how many confirmations is needed before the transaction becomes unwatched.

We are passing `spendableHeight` as `stopAfter` which means notifications will continue until each tx has a huge number of confs. We only need to wait for 1 conf I think?